src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,189 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Storage class for loading and saving files.
+ */
+public class Storage {
+
+    /** File to be loaded or saved */
+    private File file;
+
+    /**
+     * Constructor of Storage class.
+     * Create a new file if the file required is not found.
+     *
+     * @param filePath The path of file duke.txt.
+     * @throws IOException For input error.
+     */
+    public Storage(String filePath) throws IOException {
+        this.file = new File(filePath);
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+    }
+
+    public File getFile() {
+        return file;
+    }
+
+    public static String getFilePathToDukeTxt() {
+        String filePath = new File("").getAbsolutePath();
+        filePath += "/duke.txt";
+        return filePath;
+    }
+
+    /**
+     * Load the file from the txt file.
+     *
+     * @return An arraylist of tasks stored in the txt file.
+     * @throws FileNotFoundException For cases when file is not found.
+     */
+    public ArrayList<Task> loadFile() throws FileNotFoundException {
+        ArrayList<Task> ls = new ArrayList<>();
+        ArrayList<String> existing = new ArrayList<>();
+
+        //Load the current content of the file
+        Scanner s = new Scanner(getFile());
+        while (s.hasNext()) {
+            String st = s.nextLine();
+            existing.add(st);
+        }
+
+        //Turn content in the file into a list
+        for (int i = 0; i < existing.size(); i++) {
+            String tk = existing.get(i);
+            String[] arr = tk.split("<>", 4);
+
+            switch (arr[0]) {
+                case "T":
+                    try {
+                        if (Integer.valueOf(arr[1]) == 0) {
+                            //The task is not finished
+                            Task t = new ToDo(arr[2]);
+                            ls.add(t);
+                        } else if (Integer.valueOf(arr[1]) == 1) {
+                            //Task Finished
+                            Task t = new ToDo(arr[2]);
+                            t.mark();
+                            ls.add(t);
+                        } else {
+                            Ui.botUnknownFormat(i);
+                        }
+                    } catch (Exception e) {
+                        Ui.botUnknownFormat(i);
+                    }
+                    break;
```

Comment:
According to the coding standard, case clauses in switch statements should not be indented. Perhaps you can configure your IDE to follow this style?